pkgs/tools/security/magnifier/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, llvmPackages_14
+, cmake
+}:
+
+stdenv.mkDerivation rec {
+  pname = "magnifier";
+  version = "unstable-2022-08-26";
+
+  src = fetchFromGitHub {
+    owner = "trailofbits";
+    repo = "magnifier";
+    rev = "27c849db162e509aa413bdd971d7c55e32bc00a3";
+    sha256 = "sha256-yL71f3j9R52c1rFAsB9jPUoBASwdpyFeLSvSAAn9xms=";
+  };
+
+  patches = [ ./CMakeLists.txt.patch ];
```

Comment:
It would be better to delete lines altogether rather than commenting them out in the patch. Also, please rename the file to something like `no-vendored-llvm.patch` so it would be easier to guess what it does.

Reply:
good point, thanks!